### ISSUE TITLE
Is conda.recipe/meta.yml wrong?

### ISSUE THREAD

**Issue by martinholmer** [open-source-economics/OG-USA#393]
Here is what the top of the file looks like:
```
package:
  name: ogusa
  version: 0.5.10

requirements:
  build:
    - python
    - pip
    - matplotlib
    - numpy >=1.12.1
    - pandas >=0.20.1
    - scipy
    - setuptools
    - taxcalc
```
My question is how can the taxcalc package be found when doing a conda build when it's on the ospc channel and there is no specification of the ospc channel?

@hdoupe,  You explained the need for non-default channels to be specified recently in the taxdata repo.  Doesn't your recent explanation apply here?

@jdebacker @rickecon

**Comment by hdoupe** [comment]
@martinholmer Yes it does apply. I don't think there's a way to specify the channel in the `meta.yaml` file. There's an open issue on it and a few closed but not merged PR's related to it: https://github.com/conda/conda/issues/1504

This is why you have to run `conda config --add channels ospc` before running the `conda build` command.

**Comment by martinholmer** [comment]
@hdoupe said:
> Yes it does apply. I don't think there's a way to specify the channel in the `meta.yml` file. There's an open issue on it and a few closed but not merged PR's related to it: conda/conda#1504
>
> This is why you have to run `conda config --add channels ospc` before running the conda build command.

Thanks for the explanation.  I must say this conda business is pretty flaky.

**Comment by hdoupe** [comment]
No problem. The conda building and packaging interface definitely has some room for improvement.
